main/network.go

```diff
@@ -139,6 +141,8 @@ func serialOutWatcher() {
 	for i := 0; i < 10; i++ {
 		serialDevs = append(serialDevs, fmt.Sprintf("/dev/serialout%d", i))
 		serialDevs = append(serialDevs, fmt.Sprintf("/dev/serialout_nmea%d", i))
+		// Improved Serial Output support for different devices
+		serialDevs = append(serialDevs, fmt.Sprintf("/dev/ttyUSB%d", i))
```

Comment:
Not sure about this.. this is kinda a catch-all for a lot of devices, including stuff that is intended to be a GPS and not a serialout chip..
Unfortunately it's kinda impossible to guess the intention that a user has when plugging in a certain device.. can we maybe somehow filter to only display devices that are not detected as GPS? Also see my general remark though..